simplepbr/__init__.py

```diff
@@ -142,6 +142,15 @@ def __setattr__(self, name, value):
             self.tonemap_quad.set_shader_input('exposure', self.exposure)
         elif name == 'msaa_samples':
             self._setup_tonemapping()
+        elif name == 'render_node' and prev_value != value:
+            self._recompile_pbr()
+        elif name == 'camera_node' or name == 'window' and prev_value != value:
```

Comment:
`and` has a higher operator precedence than `or`, so I believe this is evaluating as:
`(name == 'camera_node)' or (name == 'window' and prev_value != value)`

This would be better written as:
`name in ('camera_node', 'window') and prev_value != value`